pkg/sass-types/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "sass-types",
```

Comment:
I'll let you decide whether you want it as `sass-types`, `sass-js-api-types`, `@sass/types` or `@sass/js-api-types` (or another name).
For now, I went for a `sass-` prefix rather than the `@sass` scope because none of the other sass packages are scoped on npm.